README.md

```diff
@@ -1 +1,24 @@
-# modular_cc_toolchains
\ No newline at end of file
+# modular_cc_toolchains
+
+This is an experimental API for a more modular Bazel toolchain.
+
+**NOTE**: Expect breaking change.
+
+## Goals
+A good rundown of why this repository exists is described in the 
+[modular cc toolchains proposal](https://docs.google.com/document/d/1-etGNsPneQ8W7MBMxtLloEq-Jj9ng1G-Pip-cWtTg_Y/edit?usp=sharing).
+
+The end goal will be to merge this into the 
+[rules_cc repository](https://github.com/bazelbuild/rules_cc). We think that we
+can implement the suggested changes with needing to make any breaking changes.
```

Comment:
*without* needing to make breaking changes!

Reply:
Oops hahaha